### DISCUSSION TITLE
How do I specify reserved instances with Gruntwork modules?

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#603] [How do I] [answered]
I need to have my databases run on reserved instances. How can I indicate this with Gruntwork modules?

---

<ins datetime="2022-11-30T17:14:11Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109675">Tracked in ticket #109675</a></p>
</ins>

**Comment by josh-padnick** [comment, marked as answer]
Reserved instances are a billing mechanism that AWS uses to determine how much to charge you for your database instances. To quote the [AWS RDS Reserved Instances marketing page](https://aws.amazon.com/rds/reserved-instances/):

> If you purchase a Reserved Instance in a given region where you have a applicable running DB instance, the benefit will automatically be applied to that instance moving forward.

Therefore, there's nothing to indicate with the Gruntwork modules!
